test/integration/sonobuoy_integration_test.go

```diff
@@ -46,15 +49,43 @@ func runSonobuoyCommandWithContext(ctx context.Context, t *testing.T, args strin
 
 	t.Logf("Running %q\n", command.String())
 
-	return command.Run(), stdout, stderr
+	return stdout, stderr, command.Run()
+}
+
+func mustRunSonobuoyCommand(t *testing.T, args string) bytes.Buffer {
+	return mustRunSonobuoyCommandWithContext(context.Background(), t, args)
+}
+
+// mustRunSonobuoyCommandWithContext runs the Sonobuoy CLI with the given context and arguments.
+// It returns stdout and fails the test immediately if there are any errors.
+func mustRunSonobuoyCommandWithContext(ctx context.Context, t *testing.T, args string) bytes.Buffer {
```

Comment:
Helper added since we already were repeatedly calling sonobuoy functions and erroring/logging on failure. This just makes the tests easier to read/write if we want to just be able to code along the happy path.

Unhappy path testing is still supported via the original methods.